docs/departments/civilian/standard-operating-procedures.mdx

```diff
@@ -4,4 +4,81 @@ sidebar_position: 1
 
 # 📃 Operating Procedures
```

Comment:
This is a the main page title. This is what shows on the menu, just so you know the syntax for other pages you may contribute to.